api/managementpb/agent.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+
+package management;
+
+import "google/api/annotations.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "api/managementpb;managementpb";
+
+message GenericAgent {
+  // Unique randomly generated instance identifier.
+  string agent_id = 1;
+  // Agent password.
+  string agent_password = 2;
+  // Agent type.
+  string agent_type = 3;
+  // AWS Access Key.
+  string aws_access_key = 4;
+  // AWS Secret Key.
+  string aws_secret_key = 5;
```

Comment:
No, there was no intent to expose it, please see https://github.com/percona/pmm/pull/1848/files#diff-30a6380a106108475836d52c52e310163f0d976cb88afa2dd6b268c1bd78cbb2R164-R172

Reply:
Maybe then we can use boolean fields instead, like `aws_access_key_present` or something like that?

Reply:
In theory we could do that as well, but I preferred to use the real db column names so that it is easier to see and control the output of all properties if anything changes. That's the only reason. And again, it's super convenient for the frontend since it doesn't need to implement additional logic to show the stars (aka placeholders) wherever there is a secret.